Docs/source/install/hpc/leonardo.rst

```diff
@@ -0,0 +1,172 @@
+.. _building-leonardo:
+
+Leonardo (CINECA)
+=================
+
+The `Leonardo cluster <https://leonardo-supercomputer.cineca.eu/>`_ is hosted at `CINECA <https://www.cineca.it/en>`_.
+
+On Leonardo, each one of the 3456 compute nodes features a custom Atos Bull Sequana XH21355 "Da Vinci" blade, composed of:
+
+* 1 x CPU Intel Ice Lake Xeon 8358 32 cores 2.60 GHz
+* 512 (8 x 64) GB RAM DDR4 3200 MHz
+* 4 x NVidia custom Ampere A100 GPU 64GB HBM2
+* 2 x NVidia HDR 2×100 GB/s cards
+
+Introduction
+------------
+
+If you are new to this system, **please see the following resources**:
+
+* `Leonardo website <https://leonardo-supercomputer.cineca.eu/>`_
+* `Leonardo user guide <https://wiki.u-gov.it/confluence/display/SCAIUS/UG3.2%3A+LEONARDO+UserGuide>`_
+
+Storage organization:
```

Comment:
Just checking: sometimes there is a recommended location to store software binaries in. There is no such one for Lenoardo, is there? (Instead of our `$HOME/sw`)

Reply:
No, I don't know of a recommended location for binaries.